react_rails_getting_started.org

```diff
@@ -320,9 +304,7 @@ class HomeController < ApplicationController
 end
 #+END_SRC
 
-Hitting http://localhost:3000/home/search_path the component rendered
-is =Components::Home::Top=.
-
+Browsing http://localhost:3000/home/search_path rendered =Components::Home::Top=.
 
```

Comment:
A note: I'm confused here; I don't know what `Components::Home::Top` is; it doesn't show previous to here.